src/Geocoder/Formatter/StringFormatter.php

```diff
@@ -51,19 +49,27 @@ class StringFormatter
      */
     public function format(Address $address, $format)
     {
-        return strtr($format, array(
+        $tr = [
             self::STREET_NUMBER => $address->getStreetNumber(),
             self::STREET_NAME   => $address->getStreetName(),
             self::LOCALITY      => $address->getLocality(),
             self::POSTAL_CODE   => $address->getPostalCode(),
             self::SUB_LOCALITY  => $address->getSubLocality(),
-            self::COUNTY        => $address->getCounty()->getName(),
-            self::COUNTY_CODE   => $address->getCounty()->getCode(),
-            self::REGION        => $address->getRegion()->getName(),
-            self::REGION_CODE   => $address->getRegion()->getCode(),
             self::COUNTRY       => $address->getCountry()->getName(),
             self::COUNTRY_CODE  => $address->getCountry()->getCode(),
             self::TIMEZONE      => $address->getTimezone(),
-        ));
+        ];
+
+        for ($level = 1; $level <= AdminLevelCollection::MAX_LEVEL_DEPTH; ++ $level) {
+            $tr[self::ADMIN_LEVEL . $level] = null;
```

Comment:
Translate... called `tr` since the function using it is `strtr` :) I'll find a better name...